bundles/cq-social-cleanup-srp/src/main/java/com/adobe/cq/social/cleanup_srp/impl/CleanupServlet.java

```diff
@@ -115,15 +115,18 @@ private void cleanup(final ResourceResolver resolver, final int batchSize, final
         long total = 0;
         boolean retried = false;
         SocialResourceConfiguration config = socialUtils.getDefaultStorageConfig();
-        SocialResource rootResource = (SocialResource) resolver.getResource(config.getAsiPath());
-        rootResource.getResourceProvider().setConfig(config);
-        rootResource.getResourceProvider().delete(resolver, path);
-        try {
-            resolver.commit();
-        } catch (PersistenceException e) {
-            LOG.info("Could not persist deletion. Maybe there is nothing to delete", e);
-        }
-
+        // For SRPs that handle recursive delete
+        if (config.getAsiPath().equals("/content/usergenerated/asi/jcr")) {
```

Comment:
Please move this string in a static variable.

Reply:
Done. Please review